tests/devices/test_default_qubit_jax.py

```diff
@@ -0,0 +1,394 @@
+import pytest
+jax = pytest.importorskip("jax", minversion="0.2")
+jnp = jax.numpy
+
+import pennylane as qml
+from pennylane.devices.default_qubit_jax import DefaultQubitJax
+

```

Comment:
You can force this test file to run in both tape and non-tape mode as so:

```suggestion

pytestmark = pytest.mark.usefixtures("tape_mode")
```